tests/test_series.py

```diff
@@ -3352,3 +3353,18 @@ def test_vector_series_normalize():
         [(x, -5, 5), (y, -5, 5), (z, -5, 5)], params={u: 1},
         slice=Plane((0, 0, 0), (0, 1, 0)), normalize=True)
     assert s.normalize
+
+def test_complex_number_eval():
+    xi, wn, x0, v0, t = symbols("xi, omega_n, x0, v0, t")
+    x = Function("x")(t)
+    eq = x.diff(t, 2) + 2 * xi * wn * x.diff(t) + wn**2 * x
+    sol = dsolve(eq, x, ics={x.subs(t, 0): x0, x.diff(t).subs(t, 0): v0})
+    d = {
+        'params' : {
+            wn: 0.5,
+            xi: 0.25,
+            x0: 0.45,
+            v0: 0.0}
+    }
+    s = InteractiveSeries([sol.rhs], [(t, 0, 100)], **d)
```

Comment:
When testing specific cases like this, it's better to also be specific about which class to instantiate: version 2 of this module might break something. In this case the appropriate class is `LineInteractiveSeries`.

Also, as you can see there are a tons of tests. It is imperative to keep them as light as possible, so that they execute as fast as possible. To achieve that we have to decrease the number of discretization points used during evaluation. So, add something like `n1=5` to the instantiation.